src/System.Runtime.Extensions/tests/VoidMainWithExitCodeApp/project.json

```diff
@@ -8,6 +8,8 @@
   },
   "frameworks": {
     ".NETCoreApp,Version=v1.0": {},
+    ".NETCoreApp,Version=v1.1": {},
+    ".NETStandard,Version=v1.5": {},
```

Comment:
I am not sure why the previous one was using the full name and I simply matched the format. I will fix it.